kalite/distributed/features/steps/superuser_create.py

```diff
@@ -5,6 +5,10 @@
 
 modal_container = "superusercreate-container"
 
+@given("I go to homepage")
```

Comment:
All the more reason not to create a superuser in that step, and instead make a new @given.